src/app/Plans/components/PlanActionsDropdown.tsx

```diff
@@ -129,7 +129,7 @@ const PlansActionsDropdown: React.FunctionComponent<IPlansActionDropdownProps> =
           </Button>,
         ]}
       >
-        <PlanDetails plan={plan} />
+        <ConnectedPlanDetails plan={plan} />
```

Comment:
I do think `ConnectedPlanDetails` needs a better name, it is reminiscent of Redux connected components which are a bit different. Maybe we could just put this `<Modal>` that is currently wrapped around it inside it, and call it `PlanDetailsModal`?

Reply:
Definitely!
Thanks for the suggestion.